docs/content/how-to/kubevirt/troubleshooting-kubevirt-cluster.md

```diff
@@ -1,5 +1,52 @@
+* If the VMs are running but they have not registered as worker nodes, use the web console to gain VNC access to one of the impacted VMs. The VNC output should indicate that the ignition config was applied. If the VM is unable to access the HCP ignition server on startup, that will prevent the VM from being provisioned correctly.
+* If the ignition config has been applied but the VM is still not registering as a node, Proceed to the [Fetching VM Bootstrap logs](#fetching vm bootstrap logs) section to gain access to the VM console logs during boot.
+
+## HCP Worker Nodes Stuck in NotReady State
+
+* During cluster creation, nodes will enter the "NotReady" state temporarily while the networking stack is rolling out. This is a part of normal operation. If this time period takes longer than 10-15 minutes, then it's possible an issue has occurred that needs investigation.
+* Look at the conditions on the node object to determine why the node is not ready. `oc get nodes -o yaml`
+* Look for failing pods within the cluster `oc get pods -A --field-selector=status.phase!=Running,status.phase!=Succeeded`
+
+## Ingress and Console cluster operators are not coming online
+
+* If the cluster is using the default ingress behavior, ensure that wildcard DNS routes are enabled on the OCP cluster the VMs are hosted on. `oc patch ingresscontroller -n openshift-ingress-operator default --type=json -p '[{ "op": "add", "path": "/spec/routeAdmission", "value": {wildcardPolicy: "WildcardsAllowed"}}]'`
+* If a custom base domain is used for the HCP, double check that the Load Balancer is targeting the VM pods accurately, and make sure the wildcard DNS entry is targeting the Load Balancer IP.
+
+## Guest Cluster Load Balancer services are not becoming available
+
+* Look for events and details associated with the Load Balancer service within the guest cluster
+* Load Balancers for the guest cluster are by default handled by the kubevirt-cloud-controller-manager within the HCP namespace. Ensure the kccm pod is online and look at the pod’s logs for errors or warnings. The kccm pod can be identified in the HCP namespace with this command `oc get pods -n <hcp namespace> -l app=cloud-controller-manager`
+
+## Guest Cluster PVCs are not available
+
+* Look for events and details associated with the PVC to understand what errors are occurring
+* If the PVC is failing to attach to a Pod, look at the logs for the kubevirt-csi-node daemonset component within the guest to gain further insight into why that might be occurring. The kubevirt-csi-node pods can be identified for each node with this command `oc get pods -n openshift-cluster-csi-drivers -o wide -l app=kubevirt-csi-driver`
+* If the PVC is unable to bind to a PV, look at the logs of the kubevirt-csi-controller component within the HCP namespace. The kubevirt-csi-controller pod can be identified within the hcp namespace with this command `oc get pods -n <hcp namespace> -l app=kubevirt-csi-driver`
```

Comment:
@awels can you take a look at this section and see if it is detailed enough. The goal here is to provide some basic steps someone could go through in order to find the root cause for why guest cluster pvcs are not becoming available or attaching to pods.